### PR TITLE
Feature/gbi 902 limit accepted quotes

### PR DESCRIPTION
**Pull request by coolwather**

### CODE REVIEW

File: http/server.go [hunk 1]
```diff
@@ -290,13 +307,26 @@ func (s *Server) getQuoteHandler(w http.ResponseWriter, r *http.Request) {
...
 	if err != nil {
 		log.Error("error decoding request: ", err.Error())
 		http.Error(w, "bad request", http.StatusBadRequest)
 		return
 	}
 	log.Debug("received quote request: ", fmt.Sprintf("%+v", qr))
 
+	maxValueTotransfer := s.cfgData.RSK.MaxQuoteValue
+
+	if maxValueTotransfer > 0 {
```
Comment (VaWheel) [inline http/server.go:320]: should it `<= 0` ?

File: main.go [hunk 2]
```diff
@@ -121,3 +124,26 @@ func main() {
...
+	if cfg.MaxQuoteValue > 0 {
+		cfgData.MaxQuoteValue = cfg.MaxQuoteValue
+	} else {
+		cfgData.MaxQuoteValue = 600000000000000000
+	}
+
+	if cfg.MaxQuoteValue > 0 {
+		cfgData.MaxQuoteValue = cfg.MaxQuoteValue
+	} else {
+		cfgData.SimultaneouslyQuotes = 1
```
Comment (VaWheel) [inline main.go:138]: why not to put it after line 132?

Comment (coolwather) [reply main.go]: Fixed

**Review by MaximStanciu8 — APPROVED** [review]

File: docker-compose/bitcoind/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM --platform=linux/amd64 ubuntu:18.04
```
Comment (manfioLP) [inline docker-compose/bitcoind/Dockerfile:1]: I think this are only used for m1 users, right?

File: docker-compose/lbc-deployer/Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM node:16
+FROM --platform=linux/amd64 node:16
```
Comment (manfioLP) [inline docker-compose/lbc-deployer/Dockerfile:1]: same

File: docker-compose/lps/Dockerfile [hunk 5]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.16-alpine
+FROM --platform=linux/amd64 golang:1.16-alpine
```
Comment (manfioLP) [inline docker-compose/lps/Dockerfile:1]: same

File: docker-compose/rskj/Dockerfile [hunk 6]
```diff
@@ -21,7 +21,7 @@ RUN gpg --keyserver https://secchannel.rsk.co/release.asc --recv-keys 1A92D89421
     ./gradlew --no-daemon clean build -x test
 
 
-FROM openjdk:8-jdk as runner
+FROM --platform=linux/amd64 openjdk:8-jdk as runner
```
Comment (manfioLP) [inline docker-compose/rskj/Dockerfile:24]: same

File: http/server.go [hunk 7]
```diff
@@ -290,13 +305,26 @@ func (s *Server) getQuoteHandler(w http.ResponseWriter, r *http.Request) {
...
 
+	maxValueTotransfer := s.cfgData.MaxQuoteValue
+
+	if maxValueTotransfer <= 0 {
+		maxValueTotransfer = s.cfgData.RSK.MaxQuoteValue
+	}
+
+	if qr.ValueToTransfer.Uint64() > maxValueTotransfer {
+		log.Error("error on quote value, cannot be greater than: ", s.cfgData.MaxQuoteValue)
+		http.Error(w, "internal server error", http.StatusInternalServerError)
```
Comment (manfioLP) [inline http/server.go:324]: maybe we should have a better msg here? I mean, I dont know if "internal server error" would be reasonable for when valueToTransfer is higher than the maximum allowed.

Maybe something like "server error: value to trransfer cant be higher than maximum allowed"?

**Review by manfioLP — APPROVED** [review]
minor comments to be confirmed

**Pull request merged**